src/@types/vscode.proposed.contribViewSize.d.ts

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+// empty placeholder for view size
+
+// https://github.com/microsoft/vscode/issues/122283 @alexr00
```

Comment:
Accidental change?

Reply:
It gets added as part of yarn. It's not needed, but I figured since everyone who yarns will get this file I might as well commit it.